packages/docusaurus-theme-live-codeblock/src/theme/CodeBlock/index.js

```diff
@@ -9,18 +9,33 @@ import React from 'react';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import usePrismTheme from '@theme/hooks/usePrismTheme';
 import Playground from '@theme/Playground';
+import ReactLiveScope from '@theme/ReactLiveScope';
 import CodeBlock from '@theme-init/CodeBlock';
+import {useMDXComponents} from '@mdx-js/react';
+
+// Returns components that are available to use by the react-live playground
+// It's not possible to import anything in the live playground,
+// we need to provide the available imports manually
+// See https://github.com/facebook/docusaurus/issues/2807
+const useReactLiveScope = () => {
+  const mdxComponents = useMDXComponents();
```

Comment:
not 100% sure this is needed as of today to provide the MDX components, as afaik we don't have a way to use the MDXProvider for now, but in the future (with wrapRootElement api) that might be a good default behavior

Reply:
Maybe we should _not_ include the built-in MDX components (`a`, `code`, `h1` etc). Does this bring any overhead?

Reply:
Actually, I tested and using in react-live `<a>` does not even pick the `a` declared in the scope, but the regular ReactDOM `a`. 

I guess it's due to their Bublé code transpilation. 

Didn't check, but it's possible that to use the `a` component available in scope, you have to do something like `React.createElement(a,props)`


If we are not so sure about providing the `useMDXComponents()` components in react-live scope, I can remove it from now and we'll see later to introduce it back if it makes sense.

Reply:
Let's better remove it, and only leave the ability to add custom components so as not to complicate this PR.